alf/examples/mbrl_pendulum.gin

```diff
@@ -0,0 +1,69 @@
+import alf.algorithms.mbrl_algorithm
```

Comment:
add this to bin/train_play_test.py for testing

Reply:
added to test